docs/install-azure-repos.md

```diff
@@ -0,0 +1,180 @@
+[Go back to the main documentation page](../README.md)
+# Installing Frogbot on Azure Repos repositories
+
+To install Frogbot on Azure Repos repositories:
+
+1. Go to Azure DevOps Pipelines of the relevant project, and add `New pipeline`.
+2. Set `Azure Repos Git` as your code source.
+3. Select the repository you'd like Frogbot to scan.
+4. Select 'Starter Pipeline' and use `frogbot-scan-pr.yml` and `frogbot-scan-and-fix.yml`:
```

Comment:
Let's replace the above section with the following two sections:

4. Select 'Starter Pipeline' and name it 'frogbot-scan-pr'. Use the content of the below yaml for the pipeline.
[expandable yml content]

5. Select 'Starter Pipeline' and name it 'frogbot-scan-and-fix'. Use the content of the below yaml for the pipeline.
[expandable yml content]

Reply:
done